pravega-cli/TestFile

```diff
@@ -0,0 +1 @@
+testing:353030303a326131393930366135663934663561633463636164636265303464386436633130353262633664663834316335363036663638373461353836363134386535353a6232333635666263313033623264303630643865613036636639656365313437633730323835333264363664343339346462626365653430353438653738623932623430336166363063303532666366383066613862626361633132643763343864353266643634356531323630613138303861303535313238653739356163:testing;
```

Comment:
What is the purpose of this file? Please, delete it if it is not necessary.

Reply:
Yes. Will remove it.